Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/inventory/MountDetailRecyclerAdapter.kt

```diff
@@ -39,7 +39,7 @@ class MountDetailRecyclerAdapter(data: OrderedRealmCollection<Mount>?, autoUpdat
     }
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MountViewHolder {
-        return MountViewHolder(parent.inflate(R.layout.animal_overview_item))
+        return MountViewHolder(parent.inflate(R.layout.pet_overview_item))

```

Comment:
Why is the `MountViewHolder` using the `pet_overview_item`? If this is reused for both, can the layout be renamed to something more fitting for how it's used?